packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -414,7 +414,8 @@ class MethodChannel {
     _methodChannelMockHandlers[this] = handler;
     binaryMessenger.setMockMessageHandler(
       name,
-      handler == null ? null : (ByteData? message) => _handleAsMethodCall(message, handler),
+      handler == null ? null // ignore: dead_code
+      : (ByteData? message) => _handleAsMethodCall(message, handler),
```

Comment:
nit: align the `:` with the `?` for clarity

Reply:
Good point.  Looks like this is moot now, since @a14n made the type of `handler` nullable.  I've reverted this part of the PR.